ckanext/harvest/logic/action/delete.py

```diff
@@ -5,30 +5,21 @@
 
 log = logging.getLogger(__name__)
 
+
 def harvest_source_delete(context, data_dict):
-    '''
-    Deletes an existing harvest source
+    '''Deletes an existing harvest source.
 
     This method just proxies the request to package_delete,
     which will delete the actual harvest type dataset and the
     HarvestSource object (via the after_delete extension point).
-
-    :param id: the name or id of the harvest source to delete
```

Comment:
Ah, got you. You would be completely right for normal python, but this is a CKAN convention for logic functions. The params are essentially the `data_dict`, and we use the docstring to document them, instead of context & data_dict themselves. More examples here: http://docs.ckan.org/en/latest/api/#ckan.logic.action.get.member_list but I completely agree that this is a non-obvious departure from python norms.